frontends/web/src/api/accountsync.ts

```diff
@@ -30,6 +30,18 @@ export const syncAccountsList = (
   return subscribeEndpoint('accounts', cb);
 };
 
+/**
+ * Returns a function that subscribes a callback on an "account/<CODE>/synced-addresses-count"
+ * event to receive the progress of the address sync.
+ * Meant to be used with `useSubscribeMap`.
+ */
+export const subscribeAddressesCount = (code: AccountCode) => (
+  (cb: TSubscriptionCallback<number>) => {
+		 return subscribeEndpoint(`account/${code}/synced-addresses-count`, (
+      count: number,
+    ) => cb(count));
+  });
+
```

Comment:
We also have `syncAddressesCount` that works slightly different (not curry function), it is a bit confusing now with both versions with `subscribeAddressesCount`. There is also other places in the app which we have curry function in combination with useSync.  I'll need some time to think about it,